ethereum/erc20/tokens.sql

```diff
@@ -4256,6 +4256,7 @@ COPY erc20.tokens (contract_address, symbol, decimals) FROM stdin;
 \\xb2D679F6D676f173fAF3670a074B2C3A6D7Ebe28	SUSDDAI	18
 \\x9B6678c766003aD69A15f795f433C0F62c10D4d5	sUSDUSDB	18
 \\x6b3595068778dd592e39a122f4f5a5cf09c90fe2	SUSHI	18
+\\x110492b31c59716ac47337e616804e3e3adc0b4a	sushiWbtcBadger 18
```

Comment:
Thank you for your contribution @bizzyvinci. There is a space instead of a tab character between `sushiWbtcBadger` and `18` . Can you please change the space into a tab? After this change, we can merge your contribution.

Reply:
Thanks, it's done.

Reply:
Thanks. I noticed the difference by checking out your branch and viewing the file in a text editor. Most editors have a way to toggle viewing whitespace details on or off. Hope this helps.